shell/browser/api/electron_api_download_item.cc

```diff
@@ -45,6 +45,16 @@ struct Converter<download::DownloadItem::DownloadState> {
   }
 };
 
+template <>
+struct Converter<download::DownloadInterruptReason> {
```

Comment:
This class can be removed since `GetLastInterruptReason` now just returns string.